ros2pkg/ros2pkg/resource/ament_cmake/package.xml.em

```diff
@@ -0,0 +1,24 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="2">
+  <name>@package_name</name>
+  <version>0.0.0</version>
+  <description>Description of @package_name</description>
+  <maintainer email="@maintainer_email">@maintainer_name</maintainer>
+  <license>Apache License 2.0</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+@[if dependencies]@
+@[  for deb in dependencies]@
+  <depend>@deb</depend>
+@[  end for]@
+
+@[end if]@
+  <test_depend>ament_lint_auto</test_depend>
+  <test_depend>ament_common</test_depend>
```

Comment:
Is `ament_common` a thing? Or is it `ament_lint_common`?

Maybe it would be better to list them individually, so it's easier to turn them on/off by just removing them or commenting them in the `package.xml`?

Reply:
> Is `ament_common` a thing? Or is it `ament_lint_common`?

It should indeed be [ament_lint_common](https://github.com/ament/ament_lint/tree/master/ament_lint_common).

> Maybe it would be better to list them individually, so it's easier to turn them on/off by just removing them or commenting them in the `package.xml`?

The advantage of depending on the metapackage is that the package will automatically benefit from linters being added in the future.

Reply:
Dunno, for our packages I think automatically getting new tests is ok, but if I were a maintainer of some higher level package, I'd probably prefer to opt into new linters rather than have them added under my feet. Either way we need a better way to opt-in/opt-out.